tests/test_NoReservedWords.py

```diff
@@ -6,7 +6,42 @@
 
 
 def check_for_issues(csc, topic):
-    jira = ""
+    if csc == "ATAOS" and topic == "Commands":
+        jira = "DM-22612"
+    elif csc == "ATMCS" and (topic == "Commands" or topic == "Events"):
```

Comment:
I am not suggesting you change the code, but in future you might consider reducing repetition using:
```
...and topic in ("Commands", "Events"):
```

Reply:
I will keep that in mind for next time.